snomed/com.b2international.snowowl.snomed.datastore.server/src/com/b2international/snowowl/datastore/server/snomed/index/SnomedServerPredicateBrowser.java

```diff
@@ -196,39 +203,62 @@ public SnomedServerPredicateBrowser(final SnomedIndexService indexService) {
 		checkNotNull(conceptId, "Concept ID must not be null.");
 
 		final Map<HierarchyInclusionType, Multimap<String, PredicateIndexEntry>> predicates = getServiceForClass(ISnomedComponentService.class).getPredicates(branchPath);
-		final HashSet<PredicateIndexEntry> newPredicates = newHashSet();
+		final Set<PredicateIndexEntry> newPredicates = newHashSet();
 
 		addPredicatesForFocus(conceptId, HierarchyInclusionType.SELF, predicates, newPredicates);
 		addPredicatesForFocus(conceptId, HierarchyInclusionType.SELF_OR_DESCENDANT, predicates, newPredicates);
 		
-		final SnomedConceptIndexEntry concept = getServiceForClass(SnomedTerminologyBrowser.class).getConcept(branchPath, conceptId);
+		final ISnomedConcept concept = SnomedRequests.prepareGetConcept()
+			.setComponentId(conceptId)
+			.setExpand("members(),relationships()")
```

Comment:
After merging https://github.com/b2ihealthcare/snow-owl/pull/151, this will require a `limit: Integer.MAX_VALUE` parameter in the `members()` expansion param.